test/parallel/test-http-get-pipeline-problem.js

```diff
@@ -35,7 +35,8 @@ const server = http.Server(function(req, res) {
 
 
 server.listen(0, function() {
-  for (var i = 0; i < total; i++) {
+  let i;
+  for (i = 0; i < total; i++) {
```

Comment:
```bash
➜  node git:(var2const) tools/test.py test/parallel/test-http-get-pipeline-problem.js 
=== release test-http-get-pipeline-problem ===                    
Path: parallel/test-http-get-pipeline-problem
image.length = 45658
/Users/gib/wrk/com/node/test/parallel/test-http-get-pipeline-problem.js:38
  for (const i = 0; i < total; i++) {
                                ^

TypeError: Assignment to constant variable.
```

If this is what you meant, I don't think so. I can do `for (let i = 0; ` though.